packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -58,6 +57,15 @@ type Props = {
      * Custom styles for the labeled field container.
      */
     style?: StyleType;
```

Comment:
We can remove the `style` prop later and move it to the `styles` object with a key for `root`. This would be a breaking change so I created https://khanacademy.atlassian.net/browse/WB-1903 for that work!

Reply:
I was thinking about that.... Do you think you could make that change in this PR? I checked in `webapp` and the `style` prop is only used once. I feel that keeping `style` and `styles` around would be a bit confusing, so it would be nicer if we could make the change right away to make the API cleaner from start.

Reply:
Sure, I can include it in these changes!